tools/tool-migration-utility/src/main/java/org/sagebionetworks/tool/migration/v3/SynapseClientFactoryImpl.java

```diff
@@ -42,7 +45,10 @@ public SynapseAdminClientImpl createNewDestinationClient() throws SynapseExcepti
 	 * @throws SynapseException 
 	 */
 	private static SynapseAdminClientImpl createNewConnection(SynapseConnectionInfo info) throws SynapseException{
-		SynapseAdminClientImpl synapse = new SynapseAdminClientImpl();
+		HttpClientProvider provider = new HttpClientProviderImpl();
+		provider.setGlobalConnectionTimeout(1000*60); 		// 	1 	min

```

Comment:
I believe you are correct:
http://hc.apache.org/httpclient-3.x/preference-api.html#HTTP_connection_parameters